cms/djangoapps/contentstore/views/course.py

```diff
@@ -768,15 +785,29 @@ def settings_handler(request, course_key_string):
                 'about_page_editable': about_page_editable,
                 'short_description_editable': short_description_editable,
                 'upload_asset_url': upload_asset_url
-            })
+            }
+            if prerequisite_course_enabled and milestones_app_enabled:
+                courses, in_process_course_actions = get_courses_accessible_to_user(request)
```

Comment:
I am having trouble following this code. Can you describe to me what the 2 return values are and how the code below ends up with "course" just being the pre_requesite_courses (on line 795)?